app/js/services/cerfaService.js

```diff
@@ -1,208 +1,102 @@
+                return ressource.type == 'indChomagePartiel';
+            });
         },
-        'cmu_c.regularite': function(individu) {
-            if ('demandeur' !== individu.role) {
-                return 'fr' !== individu.nationalite;
-            }
-
-            return false;
+        'cmu_c.vitale': function (individu) {
+            return IndividuService.age(individu) >= 18;
         },
-        'cmu_c.livret_famille': function(situation) {
-            return !! _.where(situation.individus, { role: 'enfant'}).length;
+        'cmu_c.regularite': function (individu) {
+            return individu.role != 'demandeur' && individu.nationalite != 'fr';
         },
-        'cmu_c.bulletins_paie': function(individu) {
-            if (16 > IndividuService.age(individu)) {
-                return false;
-            }
-
-            return !! _.where(individu.ressources, {type: 'revenusSalarie'}).length;
+        'cmu_c.livret_famille': function (situation) {
+            return situation.individus.some(function (individu) {
+                return individu.role == 'enfant';
+            });
         },
-        'cmu_c.attestation_indemnites_chomage': function(individu) {
-            if (16 > IndividuService.age(individu)) {
-                return false;
-            }
-
-            var nbRessourcesChomage = _.where(individu.ressources, {type: 'allocationsChomage'}).length;
-            nbRessourcesChomage += _.where(individu.ressources, {type: 'indChomagePartiel'}).length;
-
-            return !! nbRessourcesChomage;
+        'cmu_c.bulletins_paie': function (individu) {
+            return IndividuService.age(individu) >= 16 && individu.ressources && individu.ressources.some(function (ressource) {
+                return ressource.type == 'revenusSalarie';
+            });
+        },
+        'cmu_c.attestation_indemnites_chomage': function (individu) {
+            return IndividuService.age(individu) >= 16 && individu.ressources && individu.ressources.some(function (ressource) {
+                return ressource.type == 'allocationsChomage' || ressource.type == 'indChomagePartiel';
+            });
         },
-        'cmu_c.taxe_fonciere': function(situation) {
-            return 'proprietaire' === situation.logement.type;
+        'cmu_c.taxe_fonciere': function (situation) {
+            return situation.logement.type == 'proprietaire';
         },
-        'cmu_c.taxe_habitation': function(situation) {
+        'cmu_c.taxe_habitation': function (situation) {
             return _.contains(['locataire', 'colocataire'], situation.logement.type);
         },
-        'rsa.identite': function(individu) {
-            if (IndividuService.isParent(individu)) {
-                return _.contains(['fr', 'ue'], individu.nationalite);
-            }
-
-            return 'France' === individu.paysNaissance;
-        },
-        'rsa.acte_naissance': function(individu) {
```

Comment:
For children born in France but not French: we do not capture this info anymore.